src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -76,6 +78,101 @@ def chunk_iter(inputs, feature_extractor, chunk_len, stride_left, stride_right):
             yield {"is_last": is_last, "stride": stride, **processed}
 
 
+def _find_timestamp_sequence(sequences, tokenizer, feature_extractor):
+    """ """
+    max_source_positions = 1500
+    begin_idx = 3
+    last_begin_time = 0
+    items = []
+    offsets = []
+    # index of the first timestamp token
+    timestamp_begin = tokenizer.all_special_ids[-1] + 1
```

Comment:
This is wrong if a user adds any special tokens to its tokenizer.
Isn't there a more robust way to recover this particular token ?